### ISSUE TITLE
org.apache.cassandra.serializers.MarshalException: Invalid UTF-8 bytes

### ISSUE THREAD

**Issue by ashrafulsust** [Stratio/cassandra-lucene-index#94]
1. I have used cassandra 2.2.4 and cassandra-lucene-index-plugin-2.2.4.2
Let's take a table test

  ```sql
  CREATE TABLE test (
      a int PRIMARY KEY,
      b text,
      c text,
      lucene text
  )
  ```

  if we insert record as below (Explicitly specifying null for indexed column)

  ```sql
      INSERT INTO test(a , b , c ) VALUES ( 1,null,null);
  ```

  It throws the following exception :

  ```
  org.apache.cassandra.serializers.MarshalException: Invalid UTF-8 bytes 56c17ae3
  ```
2. Even if we set index column validation value false, same error occurs
3. However this error does not occur in cassandra-lucene-index-plugin-2.2.3.2
4. But set ByteBuffer as EMPTY_BYTE_BUFFER, in case of null value then it works smoothly.

**Comment by adelapena** [comment]
Hi,

We have just fixed it both in [2.2.4](https://github.com/Stratio/cassandra-lucene-index/tree/branch-2.2.4) and [2.2.5](https://github.com/Stratio/cassandra-lucene-index/tree/branch-2.2.5) branches. This fix will be available in next releases.

Thanks!

**Comment by ashrafulsust** [comment]
Thank you very much
